### DISCUSSION TITLE
"we were unable to verify your academic affiliation based on the information you submitted"

### DISCUSSION THREAD

**Discussion by guozhuyin** [community/community#52371] [GitHub Education] [answered]
I am writing to request assistance with my GitHub Education application. I have attempted to submit both my student ID and a letter of enrollment as proof of my academic affiliation, but both submissions were rejected. Additionally, I received an error message "Your uploaded proof has already been used previously."".

Could you please provide some guidance on how to proceed? I am eager to join the GitHub Education program and take advantage of its benefits, but I am unsure of how to address this issue. Any help you can provide would be greatly appreciated.
![20230410](https://user-images.githubusercontent.com/94184563/230775073-d20cacb8-7875-4d94-8c51-357e26373705.png)
![20230409](https://user-images.githubusercontent.com/94184563/230775076-d8d35b80-4249-4b00-9a7a-a70e1f52cade.png)

**Comment by samus-aran** [comment, marked as answer]
Hi @GuoZhuyin 

Let's work through those rejection reasons:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.
- It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know how you get on 👍

**Comment by guozhuyin** [comment]
Hi @samus-aran. I wanted to express my gratitude for your help in resolving my issue with the GitHub Education application. As you suggested, I realized that my previous submissions were in Chinese, and I took the time to obtain an English version of my enrollment certificate and update my personal profile accordingly. After a few attempts, my application was finally approved.

I hope that in the future, GitHub can remind users to submit documents in English to avoid confusion for non-native English speakers like myself.

Thank you for your assistance and support throughout this process.

**Comment by UsmanKhalilKIET** [comment]
Dear Sir/Madam,

Hope that you are doing well.

I am applying for Github Copilot for teachers but I am receiving an error message that the uploaded proof has already been used previously. (The picture is given below for your assistance).Kindly let me know how I can resolve this issue and apply for Github Coplit.
Thank you.

best regards,
Usman Khalil
